src/Autocomplete.js

```diff
@@ -92,6 +118,43 @@ class Autocomplete extends AutocompleteBase {
 	}
 
 	/**
+	 * Executed after List rendering.
+	 * @param {number} index
+	 * @protected
+	 */
+	handleListRender_() {
+		if (this.visible) {
+			this.currentList_ = this.element.querySelectorAll('.listitem');
+			this.activeListItem_(0);
+		}
+	}
+
+	/**
+	 * Handles a `keydown` event on this component. Handles keyboard controls.
+	 * @param {!Event} event
+	 * @protected
+	 */
+	handleKeyDown_(event) {
+		if (this.visible) {
```

Comment:
I've noticed that the only branch that's not being covered by tests right now is when this if clause fails. It would be good to have a test case making sure we don't try to do anything when a key is pressed but the autocomplete is not visible, and it would increase the coverage too, so can you add it?